src/bfb_to_raw_img.sh

```diff
@@ -124,8 +126,13 @@ bfb_basename=`basename $bfb`
 WDIR=/tmp/$bfb_basename$id
 mkdir -p $WDIR
 
+if [ "$os" == "centos" ]; then
+    cp bfb_tool_raw_img.centos.sh $WDIR/bfb_tool_raw_img.sh
+else
+    cp bfb_tool_raw_img.sh $WDIR
```

Comment:
I think we should modify bfb_tool_raw_img.sh to be called bfb_tool_raw_img.ubuntu.sh to be consistent.